### PR TITLE
Plants/set parent

### PR DESCRIPTION
**Pull request by romulus3799**
Features
- set parent plant on creation
- set parent plant on edit
- view plant's parent and offshoots in profile page
- input validation for parent
    - parent must be same species as plant
    - parent must be older than plant

### CODE REVIEW

File: src/frontend/containers/EditPlantProfilePage/index.jsx [hunk 1]
```diff
@@ -44,8 +46,19 @@ class EditPlantProfilePage extends React.Component {
       growthPictureFeedback: '',
       growthPictureValidationState: 'default',
       resetGrowthPicInput: 0,
+      errors: {
+        isBirthInvalid: false,
+        birthErrorMessage: '',
+        typeErrorMessage: '',
+        parentErrorMessage: '',
+      },
```
Comment (drewthoennes) [inline src/frontend/containers/EditPlantProfilePage/index.jsx:54]: I really appreciate that you grouped these 🙌

File: src/frontend/containers/PlantProfilePage/GeneralInformation.jsx [hunk 2]
```diff
@@ -1,74 +1,89 @@
...
+                <h4 className="text-center">{plantLocation}</h4>
+                <h5 className="text-center"><span role="img" aria-label="plantLocation">📍</span></h5>
+              </Col>
+            )}
+          </Row>
+        </Container>
+        <h5 className="mb-2 mt-3 text-center">{speciesName}, <i>{scientificName}</i></h5>
+        <h6 className="text-center">{description}</h6>
+        <br />
+        { parent && <h6 className="text-center">{name} is an offshoot of {pets[parent].name}.</h6> }
```
Comment (drewthoennes) [inline src/frontend/containers/PlantProfilePage/GeneralInformation.jsx:60]: Not necessary, but it'd be nice if the parent here was a link to that parent's profile

**Review by drewthoennes — APPROVED** [review]
Well organized and succinct!

File: src/frontend/containers/PlantProfilePage/GeneralInformation.jsx (hunk 2 above)
Comment (romulus3799) [reply src/frontend/containers/PlantProfilePage/GeneralInformation.jsx:60]: Will try to get that in the next story.

**Pull request merged**